02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,60 @@
+let {people} = require('../data')
```

Comment:
Guessing you meant to move this file to the `15-router-controller.js` file that is currently empty?